### PR TITLE
Unify timestamp format used by adapter

### PR DESCRIPTION
**Pull request by lafriks**

### CODE REVIEW

File: postgres.go [hunk 1]
```diff
@@ -22,6 +22,9 @@ import (
 	"github.com/go-rel/sql/builder"
 )
 
+// TimestampFormat used by PostgreSQL adapter.
+const TimstampFormat = "2006-01-02 15:04:05.999999999Z07:00:00"
```
Comment (Fs02) [inline postgres.go:26]: should we use the same variable name as in sql package?
https://github.com/go-rel/sql/blob/main/util.go#L11

Comment (Fs02) [reply postgres.go:26]: I feel we should not specify fixed timezone (`Z07:00:00`) here 🤔

**Review by Fs02 — COMMENTED** [review]
should we adjust

File: postgres.go (hunk 1 above)
Comment (lafriks) [reply postgres.go:26]: how do you mean fixed timezone?

Comment (lafriks) [reply postgres.go:26]: `Z07:00:00` is format for timezone (it will not change variable time zone) - see: https://play.golang.org/p/_ytFMrUOXCJ

Comment (Fs02) [reply postgres.go:26]: My bad again 🤦 (since jakarta timezine also +7)

Comment (Fs02) [reply postgres.go:26]: How about variable naming?

Comment (lafriks) [reply postgres.go:26]: oh, you mean typo? 😄

Comment (lafriks) [reply postgres.go]: Renamed constant to be same as in go-rel/sql and moved formatting to new function that could be reused also externally

**Review by Fs02 — APPROVED** [review]
LGTM 👍 👍

**Pull request merged**